test/setup-tests.js

```diff
@@ -6,6 +6,17 @@ global.window.app = {
   mcApiUrl: 'http://localhost:8080',
 };
 
+if (global.document) {
```

Comment:
This is needed for the tooltip now?

Reply:
yeah, the old __mocks__ file was deleted and replaced with this which works better with the latest version of popperjs

Reply:
OK, thanks!